tests/e2e/specs/BoundaryEventCommonBehaviour.spec.js

```diff
@@ -17,12 +17,12 @@ const taskPosition = { x: 250, y: 200 };
 const boundaryEventData = [{
   type: 'Boundary Timer Event',
   nodeType: nodeTypes.boundaryTimerEvent,
-  eventXMLSnippet: '<bpmn:boundaryEvent id="node_3" name="New Boundary Timer Event" attachedToRef="node_2" configuration="null" timing-control="null"><bpmn:timerEventDefinition><bpmn:timeDuration>PT1H</bpmn:timeDuration></bpmn:timerEventDefinition></bpmn:boundaryEvent>',
+  eventXMLSnippet: '<bpmn:boundaryEvent id="node_3" name="New Boundary Timer Event" attachedToRef="node_2"><bpmn:timerEventDefinition><bpmn:timeDuration>PT1H</bpmn:timeDuration></bpmn:timerEventDefinition></bpmn:boundaryEvent>',
   eventXMLSnippetWithoutNullAttributes: '<bpmn:boundaryEvent id="node_3" name="New Boundary Timer Event" attachedToRef="node_2"><bpmn:timerEventDefinition><bpmn:timeDuration>PT1H</bpmn:timeDuration></bpmn:timerEventDefinition></bpmn:boundaryEvent>',
```

Comment:
`eventXMLSnippetWithoutNullAttributes` should be the same as `eventXMLSnippet` and can be removed. Basically, everything I did in these commits should be reverted:
https://github.com/ProcessMaker/modeler/pull/739/commits/eecc3e6b08d50c22ee6ef58107b5020940913ba0
https://github.com/ProcessMaker/modeler/pull/739/commits/9f68ebcda23d18491e673e2501119b2adeef88e5